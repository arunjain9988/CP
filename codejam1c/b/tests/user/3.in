10
8888888
12345678
486
5999999
1000000
9850000
1000019
9999990
4648464
999989