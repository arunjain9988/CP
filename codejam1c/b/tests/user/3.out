Case #1: 9101112
Case #2: 12351236
Case #3: 567
Case #4: 7891011
Case #5: 1234567
Case #6: 9899100
Case #7: 1234567
Case #8: 10001001
Case #9: 5678910
Case #10: 1234567